python/py-resampy/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        bmcfee resampy 0.2.2
+revision            0
+name                py-${github.project}
+categories-append   audio
+
+license             ISC
+maintainers         nomaintainer
+platforms           darwin
+
+description         Efficient sample rate conversion in python.
+
+long_description    ${description}
+
+checksums           rmd160  1bfe3c1fdc94ca627008be836e0eee05d54593d5 \
+                    sha256  1500fa0ceb4e832f9d61d0d52d0e14f86919915c6a0d607fd30c1400bea0e905 \
+                    size    333571
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_run-append \
+                    port:py${python.version}-numba \
+                    port:py${python.version}-numpy \
+                    port:py${python.version}-numpydoc \
+                    port:py${python.version}-scipy \
+                    port:py${python.version}-sphinx
+
+    depends_test-append \
+                    port:py${python.version}-pytest-cov
```

Comment:
Again, probably not necessary for tests. pytest itself is usually enough.

Reply:
The requirements say it wants `pytest-cov`, so this is a copy-and-paste, but I’ll do as you prefer.

Reply:
Done thanks.